text/0093-metric-instruments.md

```diff
@@ -0,0 +1,157 @@
+# List of new metric instruments
+
+Formalize the new metric instruments proposed in [OTEP 88](https://github.com/open-telemetry/oteps/pull/88).
+
+## Motivation
+
+OTEP 88 introduced a framework for reasoning about new metric
+instruments with various refinements and ended with a [sample
+proposal](https://github.com/open-telemetry/oteps/pull/88#sample-proposal).
+This proposal uses that proposal as a starting point.
+
+Note that this proposal is meant to establish the set of standard
+instruments in terms of their refinements.  This proposal raised
+several open questions about naming and default aggregations that were
+best addressed in a separate OTEP.  See [OTEP
+96](https://github.com/open-telemetry/oteps/pull/96) and consider the
+names used in this proposal to be provisional--OTEP 96 proposes a
+final naming scheme with greater consistency.
+
+## Explanation
+
+The four instrument refinements discussed in OTEP 88 are:
```

Comment:
It seems like this document has identified 2 different categories of refinements:

- Restrictions on the input values (non-negative, precomputed-sum, and non-negative-rate). This is a refinement that acts as a contract between the instrumenter and the exporter. The former agrees to only send a type of data the latter agrees to handle/interpret the data as such.
   I imagine the unit refinement (i.e. the `Timer` related restriction) to fit in here as well, right?
- Restrictions on meaningful aggregations (sum-only). This is a refinement limiting what processing make sense to be done on the data (though I imagine you could define a view that would go against this?).

Are there any other that people see?

Did we want to explicitly include these categories in this OTEP?